Dockerfile

```diff
@@ -109,6 +109,8 @@ RUN pip install scipy
 RUN pip install git+https://github.com/chanzuckerberg/srst2
 # TODO: Test both pip installations, consider keeping pip use consistent
 RUN pip3 install pandas
+# Bedtools for obtaining total reads for each gene and calculating reads per million
+apt-get install -y bedtools
```

Comment:
It's a little surprising that there's no `RUN` here. Just verifying this is correct.

Reply:
Oops, thanks for catching that! Will fix